### ISSUE TITLE
Fix VIC inclusion strategy and ci ova biuld

### ISSUE THREAD

**Issue by morris-jason** [vmware/vic-product#353]
Opening this in response to #352.

### VIC Inclusion

Some of the installer/engine_installer code has dependencies on vic-engine, such as vic/pkg/certificate. This may increase in the future as we refactor the install to use vic-engine as a library instead of invoking exec commands on the vic-binary.

The simplest idea is to clone the vic repo to the ci container during drone builds.
```bash
# Grab the dependencies from vic if we're running ci
if [[ $DRONE ]]; then
  mkdir -p /go/src/github.com/vmware/vic/
  git clone https://github.com/vmware/vic.git /go/src/github.com/vmware/vic/
fi
```

In the future we may want to vendor the vic repo - that would be in line with the idea of the ova packaging up a specific version of the vic products. 
However, this decreases the ability make rapid changes to the ova if we have to wait on release versions of vic-engine.

It's possible we can do hybrid strategy like we currently do with Admiral and Harbor. 
Release versions of the OVA would be built with numbered (ie. 1.1.1) versions of all components using repo tags, and dev builds are pulling down the most recent containers for those products.

### OVA Build Concerns

There is also an issue with building the ova in the ci using the current build script. The build script is running in the integration-ci container, but needs SSH access to the PACKER ESX host. We need to investigate if drone can give port access to agent containers or if we need an alternative packer builder for the ci.

**Comment by andrewtchin** [comment]
@rajanashok

**Comment by morris-jason** [comment]
PR closed this, should be open.

**Comment by morris-jason** [comment]
PR #365 will add some dependencies from vic. Please make sure those are included.

**Comment by anchal-agrawal** [comment]
https://github.com/vmware/vic-product/pull/415 added some as well.

**Comment by morris-jason** [comment]
This is being addressed in #1037.
